ui/src/components/chat/Chat.tsx

```diff
@@ -336,149 +336,143 @@ export default function Chat(props: Props) {
   };
 
   return (
-    <>
-      <Paper
-        elevation={4}
+    <Paper
+      elevation={4}
+      sx={{
+        overflow: "auto",
+        width: "100%",
+        height: "100%",
+        bgcolor: "grey.300",
+        marginBottom: "2rem",
+      }}
+    >
+      <Collapse in={alert}>
+        <Alert
+          sx={{ width: "auto" }}
+          severity="error"
+          action={
+            <IconButton
+              aria-label="close"
+              color="inherit"
+              size="small"
+              onClick={() => {
+                toggleAlert(false);
+              }}
+            >
+              <CloseIcon fontSize="inherit" />
+            </IconButton>
+          }
+        >
+          {alertMsg}
+        </Alert>
+      </Collapse>
+      <Box
         sx={{
-          position: "absolute",
-          top: "50%",
-          left: "40px",
           width: "300px",
-          height: "auto",
-          bgcolor: "grey.600",
+          height: "87.5%", // TODO : responsiveness to be adapted
```

Comment:
it depends on the screen size, I didn't find a proper solution for now

Reply:
Is this why the chat has the scrollbars on the side and the bottom? Because I played around with it a bit and found a solution that is not super optimal but I think still looks good.